modules/RTC/TraceablePeerConnection.js

```diff
@@ -1035,6 +1035,33 @@ const normalizePlanB = function(desc) {
 };
 
 /**
+ * Makes sure that both audio and video directions are configured as 'sendrecv'.
+ * @param {Object} localDescription the SDP object as defined by WebRTC.
+ */
+const enforceSendRecv = function(localDescription) {
```

Comment:
makes sense but not sure i get that from the comment.  if i understand you right we're not setting it as 'sendrecv' when we actually do setLocalDescription in chrome, just making it 'sendrecv' when we get it out for sending to the other peer, yeah?  maybe the comment could be a bit clearer. (you're talking about the comment on line 1100 below, right?)